c_cpp/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && apt-get install -y --force-yes \
   && git clone https://github.com/google/googletest.git $HOME/googletest \
   && apt-get remove -y git \
   && apt-get autoremove -y \
+  && apt-get install -y wget \
```

Comment:
`wget` should be removed after installing the boost library as it's not needed for anything else. That's only useful if it's part of the same `RUN` command or it doesn't save any image space, so the boost install has to be part of the same command also.

See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/